### PR TITLE
Add the conversion of architectures for seccomp config

### PR DESCRIPTION
**Pull request by keloyang**
Add the conversion of architectures for seccomp config 
Signed-off-by: yangshukui <yangshukui@huawei.com>

### CODE REVIEW

**Comment by mheon** [comment]
Good catch on this - was meaning to do this myself, but didn't notice the spec was bumped to include architecture. +1 to getting this in.

**Comment by mrunalp** [comment]
LGTM @crosbymichael @LK4D4 PTAL

File: libcontainer/seccomp/config.go [hunk 1]
```diff
@@ -51,3 +51,33 @@ func ConvertStringToAction(in string) (configs.Action, error) {
 		return 0, fmt.Errorf("string %s is not a valid action for seccomp", in)
 	}
 }
+
+// ConvertStringToArch converts a string into a Seccomp comparison arch.
+func ConvertStringToArch(in string) (string, error) {
```
Comment (crosbymichael) [inline libcontainer/seccomp/config.go]: Couldn't this have been a map? I guess it does not matter really but usually maps are a little cleaner.

**Comment by keloyang** [comment]
@crosbymichael   thanks for your review,upated .

**Comment by crosbymichael** [comment]
@keloyang Nice work!

**Comment by crosbymichael** [comment]
LGTM

**Pull request merged**